dpe/afl/readme.md

```diff
@@ -0,0 +1,6 @@
+# Notes
```

Comment:
These notes are incomplete because I haven't run this one much. TODO: Test and complete.